metals/src/main/scala/scala/meta/internal/metals/UserConfiguration.scala

```diff
@@ -464,8 +464,8 @@ object UserConfiguration {
       getStringKey("fallback-scala-version").filter(_ != "automatic")
     val disableTestCodeLenses = {
       val isTextExplorerEnabled = clientConfiguration.isTestExplorerProvider()
-      getStringKey("test-user-interface") match {
-        case Some("Test Explorer") if isTextExplorerEnabled =>
+      getStringKey("test-user-interface").map(_.toLowerCase()) match {
+        case Some("test explorer") if isTextExplorerEnabled =>
```

Comment:
```suggestion
        case Some("test explorer") if isTestExplorerEnabled =>
```
I committed a lot of `text` typos :c

Reply:
fixed.